### ISSUE TITLE
Baritone block place scan

### ISSUE THREAD

**Issue by LysiBee** [cabaletta/baritone#2874]
• if server are laggy and baritone place blocks they can sometimes disappear, and then baritone fall into lava or whatever. Can baritone scan if block are placed or not? (Sometimes he try to replace disappeared block, but not always)

## Final checklist
- [x] I know how to properly use check boxes 
- [x] I have not used any OwO's or UwU's in this issue.

**Comment by ZacSharp** [comment]
All movements should continuously check the block to be placed and attempt to place it whenever it's not there.
This means Baritone should only fall of it's too late to replace the disappeared block.

**Comment by LysiBee** [comment]
> only fall of it's too late to replace the disappeared block.

Yup, can it be fixed? Maybe add `#stopwalkswhenplacing` or something like that

**Comment by ZacSharp** [comment]
So you want it to wait a second whenever it placed a block?
That would be horribly slow.

**Comment by LysiBee** [comment]
> So you want it to wait a second whenever it placed a block?
> That would be horribly slow.

Yes, but if this function is in baritone, then it will usually be disabled, but it is useful in some situations, for example, as I said on servers with high ping / lags

**Comment by Sumandora** [comment]
> 
> 
> So you want it to wait a second whenever it placed a block? That would be horribly slow.

you can wait for sound packets, the server only responds with a sound packet if the block placed wasnt blocked.
> use distance to get what block the sound was coming from
